commands/update.ts

```diff
@@ -147,33 +102,7 @@ ${stdout3 === '' ? stderr3 : stdout3}
                                             m.edit(`Failed to update: ${error4}`);
                                         } else {
                                             l('i', 'Synced!');
-                                            embed = new discord.MessageEmbed()
-                                                .setTitle('Update [Complete]')
-                                                .setDescription('Here is the update log!')
-                                                .addField(
-                                                    'Git Status',
-                                                    `\`\`\`
-$ git status
-
-${stdout === '' ? stderr : stdout}
-\`\`\``
-                                                )
-                                                .addField(
-                                                    'Git Add Result',
-                                                    `\`\`\`
-$ git add .
-
-${stdout2 === '' ? stderr2 : stdout2}
-\`\`\``
-                                                )
-                                                .addField(
-                                                    'Git Commit Result',
-                                                    `\`\`\`
-$ git commit -m "ProtoBot -- Update (Found uncommitted changes)"
-
-${stdout3 === '' ? stderr3 : stdout3}
-\`\`\``
-                                                )
+                                            embed
                                                 .addField(
```

Comment:
```suggestion
                                            embed.addField(
```
This looks more consistent